7.5/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:10-alpine
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV HEROKU_CLI_VERSION 7.5.11
+
+RUN apk add --no-cache --virtual=.build-deps bash
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apk add --no-cache --virtual=.persistent \
+        postgresql-client=10.4-r0
+
+RUN npm install -g heroku@"${HEROKU_CLI_VERSION}" \
```

Comment:
This is the installation method wich is strongly **not** recommended by heroku: https://devcenter.heroku.com/articles/heroku-cli#npm.

Do you use this method to be able to provide a specific version of the heroku cli?

Otherwise this installation method might be better: https://devcenter.heroku.com/articles/heroku-cli#standalone-installation

Reply:
Yes, I use this way to provide a concrete version.